portfolio/src/main/webapp/about.html

```diff
@@ -40,5 +40,134 @@
       <button onclick="addRandomQuote()">Someone great once said...</button>
       <div id="quote-container"></div>
     </div>
+
+    <!-- Thomas Hardy free CV template http://www.thomashardy.me.uk/free-responsive-html-css3-cv-template -->
+    <div id="cv" class="instaFade">
+	  <div class="mainDetails">
+		<div id="name">
+		  <h1 class="quickFade delayTwo">Leah Ryu</h1>
+		  <h2 class="quickFade delayThree">Student</h2>
+		</div>
+		
+		<div id="contactDetails" class="quickFade delayFour">
+		  <ul>
+			<li>e: <a href="mailto:ryu.leah@gmail.com" target="_blank">ryu.leah@gmail.com</a></li>
+			<li>l: <a href="https://www.linkedin.com/in/leah-ryu">Linkedin</a></li>
+			<li>g: <a href="https://github.com/nonsensicle">GitHub</a></li>
+		  </ul>
+		</div>
+		<div class="clear"></div>
+	  </div>
+	
+	  <div id="mainArea" class="quickFade delayFive">
+		<section>
+		  <article>
+			<div class="sectionTitle">
+			  <h1>Personal Profile</h1>
+			</div>
+				
+			<div class="sectionContent">
+			  <p>Computer Science/Studio Art undergraduate currently attending Dartmouth College, with two years of work experience.
+                  Aiming to combine creativity and computer skills in both professional and student roles.
+              </p>
+			</div>
+		  </article>
+		  <div class="clear"></div>
+		</section>
+		
+		<section>
+		  <div class="sectionTitle">
+			<h1>Work Experience</h1>
+		  </div>
+			
+		  <div class="sectionContent">
+			<article>
+			  <h2>STEP Intern at Google</h2>
+			  <p class="subDetails">June 2020 - Present</p>
+			  <p>Web design on an individual project and a group capstone project; communicating with peers and hosts; writing this website!</p>
+			</article>
+				
+			<article>
+			  <h2>Section Leader at Dartmouth CS</h2>
+			  <p class="subDetails">Sep 2019 - June 2020</p>
+			  <p>• Leading students enrolled in COSC01 in weekly drill sessions <br>
+                • Helping peer understand the principles of programming in Python <br>
+                • Grading coding assignments </p>
+			</article>
+				
+			<article>
+			  <h2>Women in Science Program Intern</h2>
+			  <p class="subDetails">Jan 2019 - June 2019</p>
+			  <p>• Assembling and soldering electronic components and extensively modiying and debugging Arduino code 
+                  for cold-weather rocket science experimentation.
+              </p>
+			</article>
+			</div>
+		  <div class="clear"></div>
+		</section>
+		
+		<section>
+		  <div class="sectionTitle">
+			<h1>Key Skills</h1>
+		  </div>
+			
+		  <div class="sectionContent">
+			<ul class="keySkills">
+			  <li>Python</li>
+              <li>JS/CSS/HTML</li>
+              <li>Java</li>
+              <li>Illustration</li>
+              <li>Adobe Creative Suite</li>
+              <li>Writing</li>
+			</ul>
+
+		  </div>
+		  <div class="clear"></div>
+		</section>
+        
+        <section>
+            <div class="sectionTitle">
+			  <h1>Languages</h1>
+		    </div>
```

Comment:
Need some formatting here - opening and closing tags are not lined up and we some unnecessary empty lines.